packages/cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@7val/cli",
+  "version": "0.0.0",
+  "description": "7val command line interface",
+  "main": "index",
+  "bin": {
+    "7val": "./bin/7val.js"
```

Comment:
das Programm heißt in diesem Fall `@7val/cli` wenn das gut geht ist es immernoch kein guter name
https://docs.npmjs.com/files/package.json#bin